lib/simp/rake/pupmod/helpers.rb

```diff
@@ -290,7 +290,7 @@ def custom_fixtures_hook(opts = {
     # That will give Travis a way of warning us if the changelog
     # will prevent the rpm from building.
     task :changelog_annotation, [:quiet] do |t,args|
-      warning('DEPRECATED: use pkg:create_tag_changelog')
+      warn('DEPRECATED: use pkg:create_tag_changelog')
```

Comment:
Is `:changelog_annotation` a dep of anything?  If it is, this message will probably need to be run with `rake --trace` to know what's deprecated